topydo/ui/columns/ColumnCompleter.py

```diff
@@ -0,0 +1,47 @@
+# Topydo - A todo.txt client written in Python.
+# Copyright (C) 2017 Bram Schoenmakers <bram@topydo.org>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+This module provides a completer class that can be used by the Column UI
+CommmandLineWidget.
+"""
+
+from topydo.lib.Config import config
+from topydo.ui.CompleterBase import CompleterBase, date_suggestions
+
+
+due_tag = config().tag_due()
```

Comment:
Perhaps we could cache the values, by applying memoization or using a LRU cache in the Config class. So our code can always call `config()` without worrying about performance.